acl_setup.php

```diff
@@ -352,7 +355,8 @@
 //
 $gacl->add_acl(
     array(
-        'placeholder'=>array('filler')
+        'placeholder'=>array('filler'),
```

Comment:
Remove this placeholder (this is only needed for empty acl's; btw, we did this otherwise the acl's get deleted if they are empty)